### ISSUE TITLE
Artists field on search response has a duplicated?

### ISSUE THREAD

**Issue by polvallverdu** [sigma67/ytmusicapi#437]
**Describe the bug**
A weird thing appears when obtaining search results for albums.

```json
[
  {
    "category": "Albums",
    "resultType": "album",
    "title": "DONDE QUIERO ESTAR",
    "type": "Album",
    "duration": null,
    "year": "2023",
    "artists": [
      {
        "name": "Album",
        "id": null
      },
      {
        "name": "Quevedo",
        "id": "UC7PL9aor5qNRhvhWWVXyOqA"
      }
    ],
    "browseId": "MPREb_rqH94Zr3NN0",
    "isExplicit": false,
    "thumbnails": [
      {
        "url": "https://lh3.googleusercontent.com/hzdd5VEdCJC9gfcTimpjiZO2KZAo2_dVdU_cS4enWqkfvaHQVAsRI9bbFMbQMsTKD-PXoJbQOSZo-g=w60-h60-l90-rj",
        "width": 60,
        "height": 60
      },
      {
        "url": "https://lh3.googleusercontent.com/hzdd5VEdCJC9gfcTimpjiZO2KZAo2_dVdU_cS4enWqkfvaHQVAsRI9bbFMbQMsTKD-PXoJbQOSZo-g=w120-h120-l90-rj",
        "width": 120,
        "height": 120
      },
      {
        "url": "https://lh3.googleusercontent.com/hzdd5VEdCJC9gfcTimpjiZO2KZAo2_dVdU_cS4enWqkfvaHQVAsRI9bbFMbQMsTKD-PXoJbQOSZo-g=w226-h226-l90-rj",
        "width": 226,
        "height": 226
      },
      {
        "url": "https://lh3.googleusercontent.com/hzdd5VEdCJC9gfcTimpjiZO2KZAo2_dVdU_cS4enWqkfvaHQVAsRI9bbFMbQMsTKD-PXoJbQOSZo-g=w544-h544-l90-rj",
        "width": 544,
        "height": 544
      }
    ]
  },
  {
    "category": "Albums",
    "resultType": "album",
    "title": "DQE",
    "type": "Single",
    "duration": null,
    "year": "2023",
    "artists": [
      {
        "name": "Single",
        "id": null
      },
      {
        "name": "M1",
        "id": "UCMIyN9LjvvNHGvpA2LGt4WA"
      }
    ],
    "browseId": "MPREb_C2FOPnD76i0",
    "isExplicit": false,
    "thumbnails": [
      {
        "url": "https://lh3.googleusercontent.com/P6eBkTsgnkbJQli_Qwp8KYKfKfmYnLpbftqgdcy7eTYvQ1mdl64_pMoejzFMP0malTh6ur9_ffsMAm8=w60-h60-l90-rj",
        "width": 60,
        "height": 60
      },
      {
        "url": "https://lh3.googleusercontent.com/P6eBkTsgnkbJQli_Qwp8KYKfKfmYnLpbftqgdcy7eTYvQ1mdl64_pMoejzFMP0malTh6ur9_ffsMAm8=w120-h120-l90-rj",
        "width": 120,
        "height": 120
      },
      {
        "url": "https://lh3.googleusercontent.com/P6eBkTsgnkbJQli_Qwp8KYKfKfmYnLpbftqgdcy7eTYvQ1mdl64_pMoejzFMP0malTh6ur9_ffsMAm8=w226-h226-l90-rj",
        "width": 226,
        "height": 226
      },
      {
        "url": "https://lh3.googleusercontent.com/P6eBkTsgnkbJQli_Qwp8KYKfKfmYnLpbftqgdcy7eTYvQ1mdl64_pMoejzFMP0malTh6ur9_ffsMAm8=w544-h544-l90-rj",
        "width": 544,
        "height": 544
      }
    ]
  }
]
```

**To Reproduce**
1. ```python res = ytmusic.search("dqe", "albums")```

**Additional context**

**Comment by sigma67** [comment]
It seems "Album" is being interpreted as an artist

**Comment by polvallverdu** [comment]
The "type" of the album is interpreted as an artist. If it's a single, it says Single and null

**Comment by sigma67** [comment]
closed in [40e5650](https://github.com/sigma67/ytmusicapi/commit/40e5650de3bfcb7d78b15175c82ee112cef9c87d)
